simple_homog_mpi.py

```diff
@@ -0,0 +1,133 @@
+#!/usr/bin/env python
+# 06.06.2017, c
+"""
+Solve a coupled two-scale problem in parallel. One comptational node is solving
+a macroscopic eqution while the others are solvig local microscopic problems
```

Comment:
(eqution -> equation) while the other nodes are (solvig -> solving) local microscopic problems